index.html

```diff
@@ -256,135 +256,457 @@ <h2 class="work-title">My Recent Works</h2>
+                  height="128"
+                  filterUnits="userSpaceOnUse"
+                  color-interpolation-filters="sRGB"
+                >
+                  <feFlood flood-opacity="0" result="BackgroundImageFix" />
+                  <feColorMatrix
+                    in="SourceAlpha"
+                    type="matrix"
+                    values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+                    result="hardAlpha"
+                  />
+                  <feOffset dy="16" />
+                  <feGaussianBlur stdDeviation="12" />
+                  <feColorMatrix
+                    type="matrix"
+                    values="0 0 0 0 0.145573 0 0 0 0 0.185183 0 0 0 0 0.5375 0 0 0 0.08 0"
+                  />
+                  <feBlend
+                    mode="normal"
+                    in2="BackgroundImageFix"
+                    result="effect1_dropShadow_73_257"
+                  />
+                  <feBlend
+                    mode="normal"
+                    in="SourceGraphic"
+                    in2="effect1_dropShadow_73_257"
+                    result="shape"
+                  />
+                </filter>
+                <clipPath id="clip0_73_257">
+                  <rect
+                    width="48"
+                    height="48"
+                    fill="white"
+                    transform="translate(40 24)"
+                  />
+                </clipPath>
+              </defs>
             </svg>
-            </div>
-          <div class="language-title">Skills</div>
-            <ul class="skills pro-skill">
-              <li>Codekit</li>
-              <li>Github</li>
-              <li>Codepen</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
-            </ul>
+          </div>
+          <h3 class="language-title">Skills</h3>
+          <ul class="skills pro-skill">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
         </div>
       </div>
     </section>
+    <section class="form-section">
+      <h2 class="form-text-primary">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form
+        class="form-container"
+        action="https://formspree.io/f/xrgdqjwg"
+        method="post"
+      >
+        <input
+          id="name"
+          name="name"
+          type="text"
+          class="text-field"
+          placeholder="Full name"
+          maxlength="30"
+          required
+        />
+
+        <input
+          id="email"
+          name="email-address"
+          type="email"
+          class="text-field"
+          placeholder="Email address"
+          required
+        />
+
+        <textarea
+          id="msg"
+          name="msg"
+          id="msg"
+          cols="30"
+          rows="10"
+          maxlength="50"
```

Comment:
As requested in the requirements, please make your message field accept up to 500 characters, instead of 50, for a better user experience.

![image](https://user-images.githubusercontent.com/42799579/192532948-5548929c-552d-4bee-980d-53091fd76d2e.png)